app/code/Magento/CatalogInventory/Helper/ProductStockIsQtyDecimal.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Magento\CatalogInventory\Helper;
```

Comment:
`Helper` namespace does not seem to be the best choice as it does not have any information on what it's related to. Usually it's used for classes which didn't find their place in the codebase.